README.md

```diff
@@ -1 +1,90 @@
-# functional-harness
+## testcafe-functional-harness
+
+Runs functional tests in local browsers and on SauceLabs.
+
+#### What it does
+
+1. Starts the server with the test pages.
+2. Opens the specified browsers (locally or on SauceLabs).
+3. Runs mocha tests.
+4. Closes the browsers and stops the server after all tests are finished.
+
```

Comment:
And isn't it the base URL for the site?
```
var VIEWS_PATH = '/tests';
```
If so, why *view path* - what does it mean?